MOM_input

```diff
@@ -411,20 +375,21 @@ MASS_WEIGHT_IN_PRESSURE_GRADIENT = True !   [Boolean] default = False
 ! === module MOM_hor_visc ===
 LAPLACIAN = True                !   [Boolean] default = False
                                 ! If true, use a Laplacian horizontal viscosity.
-AH = 1.0E+12                    !   [m4 s-1] default = 0.0
-                                ! The background biharmonic horizontal viscosity.
-LEITH_AH = True                 !   [Boolean] default = False
-                                ! If true, use a biharmonic Leith nonlinear eddy viscosity.
+AH_VEL_SCALE = 0.01             !   [m s-1] default = 0.0
```

Comment:
I'm happy with this, but FYI ACCESS-OM2 used `vel_micom_iso`=0 for 0.25° and 0.1°, and 0.04 ms−1 for 1°

Reply:
This may make OM3 0.25° more viscous than access-om2-025 in regions where this term dominates the viscosity (ie exceeds Smagorinsky).